package.json

```diff
@@ -22,58 +22,63 @@
     "precommit": "lint-staged"
   },
   "dependencies": {
-    "@reach/tabs": "^0.13.2",
+    "@reach/tabs": "^0.14.0",
+    "axios": "^0.21.1",
     "babel-plugin-styled-components": "^1.12.0",
     "date-fns": "^2.19.0",
-    "gatsby": "3.1.2",
+    "fast-xml-parser": "^3.19.0",
+    "gatsby": "3.2.0",
     "gatsby-background-image": "^1.5.0",
-    "gatsby-image": "3.1.0",
-    "gatsby-plugin-manifest": "3.1.0",
-    "gatsby-plugin-react-helmet": "4.1.0",
+    "gatsby-image": "3.2.0",
+    "gatsby-plugin-manifest": "3.2.0",
+    "gatsby-plugin-react-helmet": "4.2.0",
     "gatsby-plugin-robots-txt": "^1.5.5",
-    "gatsby-plugin-sharp": "3.1.2",
-    "gatsby-plugin-sitemap": "^3.1.0",
+    "gatsby-plugin-sharp": "3.2.0",
+    "gatsby-plugin-sitemap": "^3.2.0",
     "gatsby-plugin-smoothscroll": "^1.2.0",
-    "gatsby-plugin-styled-components": "^4.1.0",
-    "gatsby-plugin-typescript": "3.1.0",
-    "gatsby-source-filesystem": "3.1.0",
-    "gatsby-transformer-json": "^3.1.0",
-    "gatsby-transformer-remark": "^3.1.0",
-    "gatsby-transformer-sharp": "3.1.0",
+    "gatsby-plugin-styled-components": "^4.2.0",
+    "gatsby-plugin-typescript": "3.2.0",
+    "gatsby-source-filesystem": "3.2.0",
+    "gatsby-transformer-json": "^3.2.0",
+    "gatsby-transformer-remark": "^3.2.0",
+    "gatsby-transformer-sharp": "3.2.0",
     "modern-normalize": "1.0.0",
+    "node-fetch": "^2.6.1",
+    "polished": "^4.1.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-dropzone": "^11.3.2",
```

Comment:
we run `npx depcheck` from time to time in other projects (with some ignore patterns). might be useful for our website project too to remove unused packages.